tests/collections/nlp/test_entity_linking_model.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+import tempfile
+
+import pytest
+import wget
+from omegaconf import OmegaConf
+
+from nemo.collections.nlp.models import EntityLinkingModel
+
+
+def get_cfg(save_dir):
+    wget.download(
+        "https://raw.githubusercontent.com/vadam5/NeMo/main/examples/nlp/entity_linking/conf/umls_medical_entity_linking_config.yaml",
```

Comment:
Why are you using wget to download this file instead of having the yaml file a part of the repo?

Reply:
Shouldn't point to your own repo anyway

Reply:
about using wget, how should I specify the path to the config in a way that works with Jenkins? The unit test fails if I put the relative path to the config file from the test file. I was using the tests in tests/collections/nlp/test_nlp_exportables.py where wget is used to load the config as an example.

Reply:
@cparisien Its pointing to my own repo b/c the config file won't be in NVIDIA nemo until after the PR is merged in. There must be a better way to go about doing this.

Reply:
I decided to just define a much simpler config in the unit test, so no need to load in a config from anywhere